src/libs/events.js

```diff
@@ -20,6 +20,7 @@ const eventsList = {
   analysisEnableBeta: 'analysis:enable',
   analysisDisableBeta: 'analysis:disable',
   analysisCreate: 'analysis:create',
+  analysisImport: 'analysis:import',
```

Comment:
I added a new event here, and see in the wiki that I'll also need to add it to the mixpanel lexicon. I should do this *after* merging the PR, is that correct?